Header header
string action
string query
string response
bool fulfilled
string parameters
float32 speech_score
float32 intent_score
